### PR TITLE
Fixes to 404

### PR DESCRIPTION
**Pull request by joel-hamill**
## What are your changes?
Fixes to 404s
## What type of changes does your doc introduce?
- [x] Bug fix
- [ ] New feature 

## What is the urgency level of this change?
- [x] Blocker
- [ ] High
- [ ] Medium

## I have completed these items:
- [ ] I have tested all commands and code snippets.
- [ ] I have built locally and tested for broken links and formatting.
- [ ] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [ ] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping `@emanic`, `@sascala`, or `@joel-hamill` for reviewing pull request changes.

### CODE REVIEW

File: 1.7/usage/tutorials/cassandra/index.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ In this tutorial you will learn how to:
 
 ## Installing Cassandra
 
-Assuming you have a DC/OS cluster up and running, the first step is to [install Cassandra](https://docs.mesosphere.com/manage-service/cassandra/)
+Assuming you have a DC/OS cluster up and running, the first step is to [install Cassandra](/docs/1.8/usage/managing-services/install/)
```
Comment (karlkfi) [inline 1.7/usage/tutorials/cassandra/index.md]: This is in the 1.7 dir. Do you really want it linking to 1.8 docs?

File: 1.7/administration/installing/cloud/aws/advanced/template-reference.md [hunk 2]
```diff
@@ -160,7 +160,7 @@ The master templates ([advanced-master-1.json](https://s3.amazonaws.com/download
   </tr>
   <tr>
     <td>AcceptEULA</td>
-    <td>Read the Mesosphere EULA and indicate agreement: https://docs.mesosphere.com/community-edition-eula/.</td>
+    <td>Read the Mesosphere EULA and indicate agreement: <a href="https://docs.mesosphere.com/open-source-attribution/">https://docs.mesosphere.com/open-source-attribution/</a>.</td>
```
Comment (karlkfi) [inline 1.7/administration/installing/cloud/aws/advanced/template-reference.md]: should this link to the 1.7 version?

File: 1.8/administration/installing/cloud/aws/advanced/template-reference.md [hunk 3]
```diff
@@ -209,7 +209,7 @@ The master templates create masters, on top of the infrastructure stack already
   </tr>
   <tr>
     <td>AcceptEULA</td>
-    <td>Read the Mesosphere EULA and indicate agreement: https://docs.mesosphere.com/community-edition-eula/.</td>
+    <td>Read the Mesosphere EULA and indicate agreement: <a href="https://docs.mesosphere.com/open-source-attribution/">https://docs.mesosphere.com/open-source-attribution/</a>.</td>
```
Comment (karlkfi) [inline 1.8/administration/installing/cloud/aws/advanced/template-reference.md]: should this link directly to the 1.8 version?

File: 1.8/usage/tutorials/jenkins/index.md [hunk 4]
```diff
@@ -318,7 +318,7 @@ $ dcos package uninstall jenkins
 [azure-storage-account]: https://portal.azure.com/#create/Microsoft.StorageAccount-ARM
 [dcos-nfs-docs]: https://dcos.io/docs/1.8/administration/storage/nfs/
 [jenkins-mesos-plugin]: https://github.com/jenkinsci/mesos-plugin
-[jenkins-service-docs]: https://docs.mesosphere.com/manage-service/jenkins/
+[jenkins-service-docs]: https://docs.mesosphere.com/usage/service-guides/jenkins/
```
Comment (karlkfi) [inline 1.8/usage/tutorials/jenkins/index.md]: should this link to the 1.8 version?

File: 1.9/administration/installing/cloud/aws/advanced/template-reference.md [hunk 5]
```diff
@@ -209,7 +209,7 @@ The master templates create masters, on top of the infrastructure stack already
   </tr>
   <tr>
     <td>AcceptEULA</td>
-    <td>Read the Mesosphere EULA and indicate agreement: https://docs.mesosphere.com/community-edition-eula/.</td>
+    <td>Read the Mesosphere EULA and indicate agreement: <a href="https://docs.mesosphere.com/open-source-attribution/">https://docs.mesosphere.com/open-source-attribution/</a>.</td>
```
Comment (karlkfi) [inline 1.9/administration/installing/cloud/aws/advanced/template-reference.md]: should this link to the 1.9 version?

File: 1.9/usage/tutorials/cassandra/index.md [hunk 6]
```diff
@@ -39,7 +39,7 @@ In this tutorial you will learn how to:
 
 ## Installing Cassandra
 
-Assuming you have a DC/OS cluster up and running, the first step is to [install Cassandra](https://docs.mesosphere.com/manage-service/cassandra/)
+Assuming you have a DC/OS cluster up and running, the first step is to [install Cassandra](/docs/1.8/usage/managing-services/install/)
```
Comment (karlkfi) [inline 1.9/usage/tutorials/cassandra/index.md]: should this link to the 1.9 version?

File: 1.9/usage/tutorials/jenkins/index.md [hunk 7]
```diff
@@ -318,7 +318,7 @@ $ dcos package uninstall jenkins
 [azure-storage-account]: https://portal.azure.com/#create/Microsoft.StorageAccount-ARM
 [dcos-nfs-docs]: https://dcos.io/docs/1.9/administration/storage/nfs/
 [jenkins-mesos-plugin]: https://github.com/jenkinsci/mesos-plugin
-[jenkins-service-docs]: https://docs.mesosphere.com/manage-service/jenkins/
+[jenkins-service-docs]: https://docs.mesosphere.com/usage/service-guides/jenkins/
```
Comment (karlkfi) [inline 1.9/usage/tutorials/jenkins/index.md]: should this link to the 1.9 version?

File: 1.7/administration/installing/cloud/aws/advanced/template-reference.md (hunk 2 above)
Comment (joel-hamill) [inline 1.7/administration/installing/cloud/aws/advanced/template-reference.md]: good point. in globally replacing broken links, i wasn't necessarily considering the logic of the original :)

File: 1.9/administration/installing/cloud/aws/advanced/template-reference.md (hunk 5 above)
Comment (joel-hamill) [inline 1.9/administration/installing/cloud/aws/advanced/template-reference.md]: This line is removed in https://github.com/dcos/dcos-docs/pull/706

File: 1.8/administration/installing/cloud/aws/advanced/template-reference.md (hunk 3 above)
Comment (joel-hamill) [inline 1.8/administration/installing/cloud/aws/advanced/template-reference.md]: This line is removed in https://github.com/dcos/dcos-docs/pull/706

**Comment by karlkfi** [comment]
Follow up: https://github.com/dcos/dcos-docs/pull/716

**Pull request merged**